Holovibes/sources/api/information_api.cc

```diff
@@ -0,0 +1,16 @@
+#include "information_api.hh"
+
+namespace holovibes::api
+{
+
+#pragma region Information
+
+void start_information_display() { Holovibes::instance().start_information_display(); }
```

Comment:
missing stop_information_display functions in this file ? (see holovibes.hh)

Reply:
I want to hide worker start and stop from the user